### ISSUE TITLE
Desk.delete_case is not working

### ISSUE THREAD

**Issue by carlos4ndre** [chriswarren/desk#53]
The delete operation on cases doesn't seem to be supported and according to the README it should be.

Running as the readme example:
```
Desk.case(2096) # works, case exists
Desk.delete_case(2096) # fails, method is undefined
```

I get this error message:
```
NoMethodError: undefined method `delete_case' for Desk:Module
```

**Comment by chriswarren** [comment]
Thanks @carlos4ndre. Can you take a look at fixing it in a pull request? I won't have time to check it out soon.

**Comment by carlos4ndre** [comment]
Sure, it shouldn't be too complicated.

**Comment by chriswarren** [comment]
Fixed in #54. Thanks!
